marklogic-data-hub-central/ui/src/components/entities/mapping/mapping-step-detail/mapping-step-detail.tsx

```diff
@@ -1295,6 +1295,11 @@ const MappingStepDetail: React.FC = () => {
     setOpenStepSettings(false);
   };
 
+  const paginationOptions = {
```

Comment:
@xnikhil08 can we add this information to a new config/mapping.config.js file?

Can we also get the tableColors (line 144) from there as well?

Reply:
@wooldridge - Done.